organization.go

```diff
@@ -0,0 +1,78 @@
+package lynx
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type Organization struct {
+	ID            int64                `json:"id"`
+	Name          string               `json:"name"`
+	Address       Address              `json:"address"`
+	Email         string               `json:"email"`
+	Phone         string               `json:"phone"`
+	ForceSMSLogin bool                 `json:"force_sms_login"`
+	Parent        int64                `json:"parent"`
+	Children      []*OrganizationChild `json:"children"`
+	Notes         string               `json:"notes"`
+	Meta          map[string]string    `json:"meta"`
+	ProtectedMeta map[string]string    `json:"protected_meta"`
+}
+
+type OrganizationChild struct {
+	Name string `json:"name"`
+	ID   int64  `json:"id"`
+}
+
+type OrganizationSimple struct {
+	ID     int64  `json:"id"`
+	Name   string `json:"name"`
+	Parent int64  `json:"parent"`
+}
+
+func (c *Client) ListOrganizationSimple() ([]*OrganizationSimple, error) {
```

Comment:
If the typerrs are replaced this can be done as an argument to the ListOrganizations query